### PR TITLE
[fix #8393] Update CPG, add community hotline page

### LINKED ISSUE #8393: Create new webpage on mozilla.org for the community participation hotline content
## Description

We need to create new webpage on mozilla.org for the community participation hotline to replace https://events.mozilla.org/communityparticipationhotline, which is going away on Feb 30, 2020, and host the same content from it on the new webpage

New URL:
https://www.mozilla.org/about/governance/policies/participation/reporting/community-hotline

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] The page is live on or before February 29th, 2020

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] Splash, the hosting site for the current page, is being EOLed on February 30th, 2020

#### Tasks
- [ ] Set up URL as https://www.mozilla.org/about/governance/policies/participation/reporting/community-hotline
- [ ] Assign content (all the content and links are the same, verbatim, to that of the current page) 
- [ ] Layout should be similar but need not be exact (can conform to an existing template & Protocol components)

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Makes updates to the CPG behind the l10n tag `participation_jan2020` and adds a new community reporting hotline page.

## Issue / Bugzilla link
#8393 
#8394 
#8395 

## Testing
https://www-demo-cpg-hotline-yd2mce6ll.herokuapp.com/en-US/about/governance/policies/participation/
https://www-demo-cpg-hotline-yd2mce6ll.herokuapp.com/en-US/about/governance/policies/participation/reporting/
https://www-demo-cpg-hotline-yd2mce6ll.herokuapp.com/en-US/about/governance/policies/participation/reporting/community-hotline/

### CODE REVIEW

**Comment by craigcook** [comment]
The CPG changelog linked to doesn't seem to be live yet: https://github.com/mozilla/cpg/blob/master/CHANGELOG.md

This may be a blocker for publishing these changes. I'll chase down the right parties and see if we can resolve that today.

**Comment by craigcook** [comment]
Marking this Do Not Merge for the moment while we get signoff from D&I and sort out the changelog.

**Comment by ejregithub** [comment]
@craigcook - I passed the demo links to Emma for review.  If I don't hear from her by noon tomorrow I'll reach out again, letting her know the window is closing for any pushes before All Hands.

**Comment by craigcook** [comment]
Changelog is live now, but we've been asked to hold this for a bit while the team organizes their comms. We should get a final go-ahead this week and we can change the date right before it merges.

File: bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
 
 {% block article %}
 <h1 class="mzp-c-article-title">{{ _('Mozilla Community Participation Guidelines') }}</h1>
-{% if l10n_has_tag('participation_nov2018') %}
-  <h4>{{ _('Version 3.0 – Updated November 28, 2018') }}</h4>
+{% if l10n_has_tag('participation_jan2020') %}
+  <h4>{{ _('Version 3.1 – Updated January 15, 2020') }}</h4>
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html]: Note: we'll need to change this date to the correct publishing date once we have final signoff.

File: bedrock/mozorg/templates/mozorg/about/governance/policies/reporting.html [hunk 2]
```diff
@@ -57,7 +57,7 @@ <h2>{{ _('How to Give a Report') }}</h2>
   </p>
 
   <p>
-  {% trans community_hotline='https://events.mozilla.org/communityparticipationhotline' %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/reporting.html]: Not related to this change, but the blockquote on this page feels a little overpowering. Worth adding a custom style to make it a bit less in-your-face?

image

Comment (craigcook) [reply bedrock/mozorg/templates/mozorg/about/governance/policies/reporting.html]: Yeah, it's a bit much, especially on such a lengthy chunk of text. We should revisit this in Protocol but for now I've added a custom style just for this page.

**Pull request merged**